### DISCUSSION TITLE
"Dislike" a show

### DISCUSSION THREAD

**Discussion by summeranne** [javikr/sofatime#27] [Ideas]
I would love to see a feature added where I can "Dislike" a show, over and above rating it 1 star.  This way I can still track that I've watched part of it, but future episodes will not show in my Watching list.  And for shows that I haven't started that I KNOW I would never like, "dislike" to remove them from being constantly suggested in the recommendations just because they are popular or watched by people with similar interests.  

Hulu has the ability to dislike and when you do: "We won’t recommend this to you anymore and we’ll use this to improve your recommendations", and I love it!

**Comment by javikr** [comment]
Hi!

Now you can “discard” tv shows that you have started to watch. Just tap on the watched button and then select “Discard”.

Discarded shows won’t show anymore in your watching and next to watch lists. Also you can also access to all your discarded shows from your profile screen.

Hope this helps ;)
